src/scoped.component.test.js

```diff
@@ -8,30 +8,6 @@ describe("<Scoped />", function() {
     resetCounter()
   })
 
-  it("should generate and cleanup style tags", function() {
-    expect(document.querySelectorAll(`style[type="text/css"]`).length).toBe(0);
-    const css = `
-      & .someRule {
-        background-color: red;
-      }
-    `;
-
-    const el = document.createElement("div");
-
-    ReactDOM.render(
-      <div>
-        <Scoped css={css}>
-          <div>Hello</div>
-        </Scoped>
-      </div>,
-      el
-    );
-    expect(document.querySelectorAll(`style[type="text/css"]`).length).toBe(1);
-
-    ReactDOM.unmountComponentAtNode(el);
-    expect(document.querySelectorAll(`style[type="text/css"]`).length).toBe(0);
-  });
-
```

Comment:
Any particular reason this test was removed? As far as I can tell this test could still be useful, the replacement tests overlap but don't completely cover this case.